constants.py

```diff
@@ -2,7 +2,7 @@
 
 import os
 
-root = os.path.expanduser("~/.sentry-dev")
+root = os.path.expanduser("~/.config/sentry-dev")
```

Comment:
i ultimately decided against this because this is the root where we put everything (pythons, virtualenvs, etc.), and ~/.config should be strictly for configuration files only